nomenclature/definitions/subannual/README.md

```diff
@@ -43,6 +43,59 @@ with open(f'../subannual/months.yaml', 'r') as stream:
 mapping = dict([(m, eval(attr['duration'])) for (m, attr) in months.items()])
 ```
 
+### Daily and hourly data
+Our convention makes reference to the following agreements:
+
+
+#### UTC datetime format:
+- Standard datetime format     : **UTC**  
+> The UTC datetime format is adopted to consider the following levels: year, month, day, hour, minute, and second (without consider time zones).
+> For example: `2020-01-01, 2020-01-02, ...` or `2020-01-01T13:00, ...`
+> The use of lists of dates and datetimes will be generated by the codes of the next section.
+- Using time zone                    : **no relevance**
+
+
+#### Using the format:
+
+The code snippet (Python) below shows how to get lists of dates and datetimes.
+
+- To get a list of dates:
+```python
+from datetime import timedelta, date
+
+def daterange(date1, date2):
+    for n in range(int ((date2 - date1).days)+1):
+        yield date1 + timedelta(n)
+
+start_dt = date(2015, 12, 20)
+end_dt = date(2016, 1, 11)
+for dt in daterange(start_dt, end_dt):
+    print(dt.strftime("%Y-%m-%d"))
+```
+- To get a list of datetimes:
+```python
+from datetime import timedelta, datetime
+
+def DateTimeRange(datetime1, datetime2):
+    for n in range(int (((datetime2 - datetime1).days)+1)*24):
+        yield datetime1 + timedelta(hours=n)
+
+start_dt = datetime(2020, 6, 23)
+end_dt   = datetime(2020, 6, 30)
+for dt in DateTimeRange(start_dt, end_dt):
+    print(dt.strftime("%Y-%m-%d %H:%M:%S"))
+```
+
+The format `"%Y-%m-%d %H:%M:%S"` is composed by tokens. Each token represents a different part of the date-time, like day, month, year, etc. More details can be found in [strftime() and strptime() section](https://docs.python.org/3/library/datetime.html).
```

Comment:
There is no such section yet, right?

Reply:
Yeah, right. It should be written "strftime() and strptime() Format Codes".